### PR TITLE
Incoming tasks for every Developer

### PR DESCRIPTION
**Pull request by AliHanifdev1**

### CODE REVIEW

**Comment by AliHanifdev1** [comment]
@MKRNaqeebi  kindly review

File: src/components/FortyHours.vue [hunk 1]
```diff
@@ -17,6 +17,11 @@
           </li>
         </ul>
       </div>
+       <ul>
+        <li v-for="(tempassignee, assignee) in tempassignee" v-bind:key="assignee">
+           {{ assignee }}: total Hours: {{tempassignee }}
```
Comment (MKRNaqeebi) [inline src/components/FortyHours.vue]: you do not need new UL for that use above one

Comment (MKRNaqeebi) [reply src/components/FortyHours.vue]: you need to write it in same <ul><li> as current, previous

**Review by MKRNaqeebi — CHANGES_REQUESTED** [review]

File: src/components/FortyHours.vue (hunk 1 above)
Comment (AliHanifdev1) [reply src/components/FortyHours.vue]: done

File: src/components/FortyHours.vue [hunk 2]
```diff
@@ -13,7 +13,7 @@
         <ul>
           <li v-for="(hour, assignee) in hours" v-bind:key="assignee">
             {{ assignee }}: current: {{ hour.current }} and previous:
-            {{ hour.previous }} hours
+            {{ hour.previous }} hours total Hours: {{hour.totalHours }}
```
Comment (MKRNaqeebi) [inline src/components/FortyHours.vue:16]: Change it to
```
<li v-for="(hour, assignee) in hours" v-bind:key="assignee">
            {{ assignee }}:
            Current: {{ hour.current }} hours<br/>
            Previous: {{ hour.previous }} hours<br/>
            Incomming: {{hour.incomming }}
</li>
```

**Review by MKRNaqeebi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**